src/plugins/client-renderer.js

```diff
@@ -10,7 +10,7 @@ import type {Context} from '../types.js';
 
 export default function createClientRenderer({render}: {render: any}) {
   return function renderer(ctx: Context, next: () => Promise<void>) {
-    const rendered = render(ctx.element);
+    const rendered = render(ctx.element, ctx);
```

Comment:
If it's possible to rebase this against master so we don't see this change in this PR that would be ideal.